openfed.install

```diff
@@ -93,3 +94,36 @@ function openfed_update_101206(&$sandbox) {
     $installer->install(['twig_real_content']);
   }
 }
+
+/**
+ * Implements hook_update_dependencies().
+ */
+function openfed_update_dependencies() {
+  $dependencies = [];
+
+  if (Drupal::moduleHandler()->moduleExists('yoast_seo')) {
+    $dependencies['openfed'][120301] = [
+      'yoast_seo' => 8204,
+    ];
+  }
+
+  return $dependencies;
+}
+
+/**
+ * Disable the yoast_seo auto refresh feature for existing websites.
+ */
+function openfed_update_120301(&$sandbox) {
+  if (!Drupal::moduleHandler()->moduleExists('yoast_seo')) {
+    return t('Yoast SEO is not enabled. No changes applied.');
```

Comment:
@ozmodiar ,
While it's great to provide feedback, I think we don't need to be too verbose.
Feedback is welcome in some cases, like how many items were updated or so, but in this case, I think we can just keep the standard "update hook has been executed".

I'll merge this, but this is something to keep in mind. ;)